logviewer/logviewer-lib.pl

```diff
@@ -138,5 +240,12 @@ sub extra_log_files
 return @rv;
 }
 
+# config_post_save
+# Called after the module's configuration has been saved
+sub config_post_save
```

Comment:
What has saving the config got to do with the cache?

Reply:
You're right now it doesn't have much to do with module config, but originally it did, as the extra units were defined there before I deleted it.

I think saving config and updating cache isn't horrible idea. It makes sense. But I will implement separate buttons for this in the new PR later tomorrow.

Reply:
A button to clear the cache? That feels like unnecessarily exposing internal details to the user..

Reply:
Agreed.